string leader
string follower1
string follower2
string follower3
string follower4
uint8 formation